src/execution_plan/ops/op_aggregate.c

```diff
@@ -242,6 +242,18 @@ static OpResult AggregateReset(OpBase *opBase) {
 	return OP_OK;
 }
 
+static OpBase *AggregateClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_AGGREGATE);
+	OpAggregate *op = (OpAggregate *)opBase;
+	uint aggregate_count =  op->aggregate_count;
+	uint key_count = op->key_count;
+	uint i;
```

Comment:
I feel like these local assignments are unnecessary, but it's not particularly important!